impeller/entity/contents/filters/filter_contents.h

```diff
@@ -125,10 +125,22 @@ class FilterContents : public Contents {
       bool msaa_enabled = true,
       const std::string& label = "Filter Snapshot") const override;
 
+  // |Contents|
+  const FilterContents* AsFilter() const override;
+
   virtual Matrix GetLocalTransform(const Matrix& parent_transform) const;
 
   Matrix GetTransform(const Matrix& parent_transform) const;
 
+  /// @brief  Returns true of this filter graph performs basis transformations
+  ///         to the filtered content. For example: Rotating, scaling, and
+  ///         skewing are all basis transformations, but translating is not.
```

Comment:
Is translating the only kind of non-basis transform? Could we rephrase this as "TranslateOnly"?

Reply:
Yeah, inverted this.